packages/application-shell/src/components/login-sso-callback/login-sso-callback.js

```diff
@@ -41,7 +41,7 @@ export class LoginSSOCallback extends React.PureComponent {
   };
 
   componentDidMount() {
-    const fragments = qs.parse(this.props.location.hash.substring(1));
+    const fragments = decode(this.props.location.hash.substring(1));
```

Comment:
This is needed otherwise the object has `{ '?foo': 'bar, ... }` which is not bad but not what we want here.